### ISSUE TITLE
Copy or Fork

### ISSUE THREAD

**Issue by cowen71** [victorclaessen/mqtt-LabVIEW#1]
Hi victorclaessen,
Unless I am mistaken, this driver development is based on my driver development. 
https://github.com/cowen71/mqtt-LabVIEW

Did you mean to copy this or rather create a fork for your further developments? Currently your modification and improvements are not feeding back to the original development.
Gruß
cowen71

**Comment by victorclaessen** [comment]
Hi @cowen71,

Sorry for the late reply, I was on holiday. 

For sure, this is a copy of your work (and by no means did I mean to steal your thunder). I was working on it from a private repository, and uploaded it to github when Steve asked for updates (https://github.com/cowen71/mqtt-LabVIEW/issues/1).

What is, in your opinion, the best way to feed these modifications back to your repo?

Best regards,

Victor

**Comment by cowen71** [comment]
Hi Victor,
I have no problem at all that you are developing this further. That is the beauty of the open community and Github.

I have a friend that works at GitHub. I have to ask them how to proceed actually. I know LabVIEW well, GitHub is a new friend!

Thanks for the response and Kind Regards

**Comment by victorclaessen** [comment]
Yeah from the code I already gathered that you knew a little something about LabVIEW ;-) Great work!
